image/base/install-imagemagick

```diff
@@ -17,7 +17,7 @@ cd $WDIR/libpng
 make all && make install
 
 # Build and install ImageMagick
-wget -O $WDIR/ImageMagick.tar.gz "http://www.imagemagick.org/download/ImageMagick.tar.gz"
+wget -O $WDIR/ImageMagick.tar.gz "http://www.imagemagick.org/download/ImageMagick-6.9.4-1.tar.gz"
```

Comment:
ImageMagick-7 has been released and rebuilding a new image resulted in ImageMagick being upgraded to 7.